WordPressKitTests/BlockEditorSettingsServiceRemoteTests.swift

```diff
@@ -1,23 +1,27 @@
 import XCTest
+import OHHTTPStubs
 @testable import WordPressKit
 
 class BlockEditorSettingsServiceRemoteTests: XCTestCase {
-    private let blockSettingsNOTThemeJSONResponseFilename = "wp-block-editor-v1-settings-success-NotThemeJSON"
-    private let blockSettingsThemeJSONResponseFilename = "wp-block-editor-v1-settings-success-ThemeJSON"
-    private let twentytwentyoneResponseFilename = "get_wp_v2_themes_twentytwentyone"
+    private let blockSettingsNOTThemeJSONResponseFilename = "wp-block-editor-v1-settings-success-NotThemeJSON.json"
+    private let blockSettingsThemeJSONResponseFilename = "wp-block-editor-v1-settings-success-ThemeJSON.json"
+    private let twentytwentyoneResponseFilename = "get_wp_v2_themes_twentytwentyone.json"
     private let testError = NSError(domain: "tests", code: 0, userInfo: nil)
     private let siteID = 1
 
     private var service: BlockEditorSettingsServiceRemote!
-    var mockRemoteApi: MockWordPressComRestApi!
 
     override func setUp() {
-        mockRemoteApi = MockWordPressComRestApi()
-        service = BlockEditorSettingsServiceRemote(remoteAPI: mockRemoteApi)
+        super.setUp()
+        stub(condition: { _ in true }) {
+            XCTFail("Unexpected request: \($0)")
+            return HTTPStubsResponse(error: URLError(URLError.Code.networkConnectionLost))
+        }
+        service = BlockEditorSettingsServiceRemote(remoteAPI: WordPressComRestApi())
     }
 
     func mockedData(withFilename filename: String) -> AnyObject {
-        let json = Bundle(for: BlockEditorSettingsServiceRemoteTests.self).url(forResource: filename, withExtension: "json")!
+        let json = Bundle(for: BlockEditorSettingsServiceRemoteTests.self).url(forResource: filename, withExtension: nil)!
```

Comment:
Just curious why you decided to add the JSON extension to the file names and use `nil` here rather than keeping the code as it was?

To be clear, I like it. It makes it clearer at the filename `let` level what extension it is. We have little reason to expect to be using files with other extensions in the future, so we might as well keep the information (name + extension) all in the same place.

Reply:
Using full filename means we can reuse the constants in here and the `OHPathForFile` calls.